app/tree/gui/package.json

```diff
@@ -7,7 +7,9 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.44",
     "@tetherless-world/twxplore-base-lib": "1.0.1",
+    "@types/graphql": "^14.5.0",
```

Comment:
Not sure why this was modified?

Reply:
Probably something I did in desperation while fixing type/import errors before copying in graphql.d.ts